### PR TITLE
feat(material-experimental/mdc-snackbar): Add skeleton.

### PR DESCRIPTION
**Pull request by opozo**
Create snackbar skeleton based on MDC Web

### CODE REVIEW

**Comment by opozo** [comment]
@jelbourn Not sure if you want me to create a PR for each steps like is in the doc or just create a single PR with all the steps, let me know.

**Comment by jelbourn** [comment]
Separate steps are definitely better

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -103,6 +103,7 @@
 /src/material-experimental/mdc-progress-bar/**     @andrewseguin
 # Note to implementer: please repossess
 /src/material-experimental/mdc-radio/**            @mmalerba
+/src/material-experimental/mdc-snackbar/**         @opozo
```
Comment (jelbourn) [inline .github/CODEOWNERS]: In your next PR, you can add me here as well

**Comment by jelbourn** [comment]
Go ahead and comment on this PR once you have all the CI stuff passing. Let me know if you need any help.

File: .github/CODEOWNERS (hunk 1 above)
Comment (opozo) [reply .github/CODEOWNERS]: Ack.

**Comment by opozo** [comment]
> Go ahead and comment on this PR once you have all the CI stuff passing. Let me know if you need any help.

@jelbourn, just came back from vacation, all the CI stuff are now passing.

**Pull request merged**